### DISCUSSION TITLE
Set Access to A Model

### DISCUSSION THREAD

**Discussion by smsp2029** [spatie/laravel-permission#2260] [Q&A]
Hi guys.
I want to set role access to a model.
For example, I have 2 Roles. An Admin and a User.
And I have category A, B and C.
Admin can access all categories, But User can only access selected category from admin. (For example A and C only)

**Comment by angeljqv** [comment]
Sounds like teams, take a look if that's what you're looking for
https://spatie.be/docs/laravel-permission/v5/basic-usage/teams-permissions
